dev-python/pypubsub/pypubsub-4.0.3.ebuild

```diff
@@ -0,0 +1,26 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+PYTHON_COMPAT=( python3_{5,6,7} )
+
+inherit distutils-r1
+
+DESCRIPTION="A Python publish-subcribe library"
+HOMEPAGE="https://github.com/schollii/pypubsub/"
+SRC_URI="https://github.com/schollii/pypubsub/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="BSD"
+SLOT="4"
+KEYWORDS="~amd64 ~x86"
+IUSE="test"
+
+DEPEND="test? ( dev-python/pytest[${PYTHON_USEDEP}] )"
```

Comment:
There is now a function in distutils-r1.eclass that can handle test deps for you. Check distutils_enable_tests